recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -124,28 +295,83 @@
    "source": [
     "## Adding a System Prompt\n",
     "\n",
-    "A system prompt can be used to provide additional instructions and clarity or context for the task. Here we let the model know what we expect from it."
+    "A system prompt can be used to provide additional instructions and clarity or context for the task. Here we let the model know what we expect from it.\n",
+    "\n",
+    "Let's define a final helper function that uses a system prompt.\n",
+    "\n",
+    "> **NOTE:** Try moving the `Make sure you only generate code that is {os_name()}-compatible!` from the user prompt to the system prompt. In our experiments, it seemed to be ignored sometimes when in the system prompt."
```

Comment:
This is interesting. Should we figure out how to get it to work from the system prompt?

Reply:
Yes, this gets moved to the system prompt in the last example, when system prompts are introduced.